astroquery/cadc/core.py

```diff
@@ -11,17 +11,26 @@
 import warnings
 import requests
 from numpy import ma
+from six.moves.urllib.parse import urlencode
 
 from ..utils.class_or_instance import class_or_instance
 from ..utils import async_to_sync, commons
 from ..query import BaseQuery
 from bs4 import BeautifulSoup
-from six import BytesIO
-from astropy.io.votable import parse_single_table
-from astroquery.cadc.cadctap.core import TapPlusCadc
-from astroquery.cadc.cadctap.job import JobCadc
+from astropy.utils.exceptions import AstropyDeprecationWarning
+from astroquery.utils.decorators import deprecated
 from . import conf
 
+try:
+    import pyvo
+except ImportError:
+    print('Please install pyvo. astropy.cadc does not work without it.')
```

Comment:
maybe this should be an import error too, but with this message?

Reply:
At the moment, `pyvo` is an optional package in `astroquery`. Users of `astroquery.cadc` will need to manually install it. What other message do you suggest?

Reply:
the text is fine, but I think this could be raised as an Exception since pyvo is not an optional dependency of the CADC module.

Reply:
Had that originally but couldn't run the tests without `pyvo` installed even if I tried to skip them in that case. Something in the `astropy pytest` environment hits the `astroquery.cadc` package during test collection and fails before I had a chance to deal with this in the test files.

Reply:
let's open an issue about this. Actually the thing we thought being an issue is an expected pytest behaviour. They might add a new feature on top of that, but that won't happen any time soon.